File: GenesisAndroid/src/main/java/com/emerchantpay/gateway/genesisandroid/api/constants/Environments.java
package com.emerchantpay.gateway.genesisandroid.api.constants;

import java.io.Serializable;

public class Environments implements Serializable {

	private String environmentName;

	// Production Environments
	public static Environments PRODUCTION = new Environments("gate");

	// Staging Environments
	public static Environments STAGING = new Environments("staging.gate");

	public Environments(String environmentName) {

		this.environmentName = environmentName;
	}

	public String getEnvironmentName() {
		return this.environmentName;
	}

	public String toString() {
		return getEnvironmentName();
	}
}

File: GenesisAndroid/src/main/java/com/emerchantpay/gateway/genesisandroid/api/constants/Environments.kt
package com.emerchantpay.gateway.genesisandroid.api.constants

import java.io.Serializable

class Environments(val environmentName: String) : Serializable {

    override fun toString(): String {
        return environmentName
    }

    companion object {

        // Production Environments
        val PRODUCTION = Environments("gate")

        // Staging Environments
        val STAGING = Environments("staging.gate")
    }
}
